docs/ingestion/input-sources.md

```diff
@@ -462,6 +465,7 @@ Sample specs:
 |--------|-----------|-------|---------|
 |type|Set the value to `hdfs`.|None|yes|
 |paths|HDFS paths. Can be either a JSON array or comma-separated string of paths. Wildcards like `*` are supported in these paths. Empty files located under one of the given paths will be skipped.|None|yes|
+|systemFields|JSON array of system fields to return as part of input rows. Possible values: `__file_uri` (URI) and `__file_path` (path component of URI).|None|no|
```

Comment:
nit: a note on the hdfs uri scheme similar to the other input formats:
```suggestion
|systemFields|JSON array of system fields to return as part of input rows. Possible values: `__file_uri` (URI starting with `hdfs://`) and `__file_path` (path component of URI).|None|no|
```

Reply:
Ah. It won't always start with `hdfs`, is the thing. HDFS input source can also read files, S3, etc, depending on the value of `druid.ingestion.hdfs.allowedProtocols` (discussed in the paragraph following this table).

Reply:
Oh, I see, thanks for the clarification